packages/usa-button/src/usa-button.stories.js

```diff
@@ -51,6 +70,13 @@ Base.args = BaseContent;
 export const Big = Template.bind({});
 Big.args = BigContent;
 
+export const Icon = Template.bind({});
+Icon.args = {
+  ...DefaultContent,
+  icon_included: true,
+  icon_name: "add_circle_outline",
```

Comment:
Explicitly setting it will preselect it in the control, which I think is a good feature. Otherwise it will **still** show the icon, but it won't be selected. Added a note in 3c3684e.

### With `icon_name`
<img width="500" alt="image" src="https://github.com/uswds/uswds/assets/3385219/c13b2813-392d-46cf-821d-fae4cc563284">


### Without `icon_name`
<img width="500" alt="image" src="https://github.com/uswds/uswds/assets/3385219/8aa3214c-1ae2-49a6-a862-cb9e574c173b">